src/components/CreateTask.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="wrap">
+    <div class="title-wrap">
+      <button
+        class="remove-list-btn"
+        @click="onClickRemoveListBtn"
+        type="button"
+      >Remove list</button>
+      <h2> {{ todoList.listTitle }} </h2>
+    </div>
+    <form v-on:submit.prevent="createTask">
+      <input
+        type="text"
+        v-model="taskText"
+        placeholder="What needs to be done?"
+      >
+      <button>Create</button>
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'CreateTask',
+  data: () => ({
+    taskText: '',
+  }),
+  methods: {
+    createTask() {
+      this.$emit('getTask', {
+        taskText: this.taskText,
+        listId: this.todoList.listId,
```

Comment:
You don't need to emit **listId** property because you can get it in parent component